definitions/json/EnglandWales Scrubbed/EnglangWales-Scrubbed-Bundles.json

```diff
@@ -31,20 +31,20 @@
       },
       {
         "ID": "frl_bundlesWhatDocuments",
-        "ListElement": "Hearing documents, including witness statements",
-        "ListElementCode": "Hearing documents, including witness statements",
+        "ListElement": "Documents that will be referred to by witnesses when giving evidence, sometimes called the \"hearing bundle\"",
+        "ListElementCode": "Documents that will be referred to by witnesses when giving evidence, sometimes called the \"hearing bundle\"",
         "DisplayOrder": 1
       },
       {
         "ID": "frl_bundlesWhatDocuments",
-        "ListElement": "Supplementary hearing documents",
-        "ListElementCode": "Supplementary hearing documents",
+        "ListElement": "Witness statements (only upload these if the tribunal has said witness statements are to be used)",
+        "ListElementCode": "Witness statements (only upload these if the tribunal has said witness statements are to be used)",
         "DisplayOrder": 2
       },
       {
         "ID": "frl_bundlesWhatDocuments",
-        "ListElement": "Witness statements only",
-        "ListElementCode": "Witness statements only",
+        "ListElement": "A combined hearing bundles and witness statements",
+        "ListElementCode": "A combined hearing bundles and witness statements",
```

Comment:
@cindychmcts the ListElementCode will have to stay as the original as these codes are live and might be used in production cases. The ListElement can change but the ListElementCode should be what it was previously. There still is a 1-1 mapping of the new and old so should be ok